test/test_trixi.jl

```diff
@@ -130,12 +130,30 @@ macro test_nowarn_mod(expr)
 end
 
 
+"""
+    @timed_testset "name of the testset" #= code to test #=
+
+Similar to `@testset`, but wraps the execution of the testset using `@time`
+and prints the name of the testset after execution.
+"""
+macro timed_testset(name, expr)
+  @assert name isa String
+  quote
+    @time @testset $name $expr
+    flush(stdout)
+    @info("Testset " * $name * " finished.\n")
+    flush(stdout)
```

Comment:
This assumes both `@time` and `@info` write to `stdout`. Is this a restriction relevant in practice? Probably not, I just wanted to be sure...

Reply:
I can't really think of a situation where this is not the case - at least given our current CI setup. Thus, I think it's okay to use it as it is.